staging/src/k8s.io/controller-manager/pkg/clientbuilder/client_builder.go

```diff
@@ -57,11 +60,13 @@ type SimpleControllerClientBuilder struct {
 	ClientConfig *restclient.Config
 }
 
+// Config returns a client config for a fixed client
 func (b SimpleControllerClientBuilder) Config(name string) (*restclient.Config, error) {
 	clientConfig := *b.ClientConfig
 	return restclient.AddUserAgent(&clientConfig, name), nil
 }
 
+// ConfigOrDie returns a client config if no error from previous config func. Otherwise it will fail with error
```

Comment:
Better to be explicit. "If it gets an error getting the client it will log the error and then kill the process its running in."

Reply:
Updated the comment. Would you please take a look again? Thank you